src/libaktualizr/storage/invstorage.cc

```diff
@@ -25,7 +25,14 @@ void StorageConfig::updateFromPropertyTree(const boost::property_tree::ptree& pt
 }
 
 void StorageConfig::writeToStream(std::ostream& out_stream) const {
-  writeOption(out_stream, type, "type");
+  std::string storage_type;
```

Comment:
Good catch!

Would it work if we moved [this function](https://github.com/advancedtelematic/aktualizr/blob/4bdb35dfbc96351ffd1e1ef9345cbb20966766e1/src/libaktualizr/config/config.cc#L19) to this file?

Reply:
Ah yes, that was presumably supposed to be what made that work. Let me give that a try.

Reply:
Done. Even better catch. Thanks!